### DISCUSSION TITLE
Disable code hightlight only

### DISCUSSION THREAD

**Discussion by alfredomtx** [mark-wiemer/ahkpp#360] [Q&A] [answered]
Hi! Is there a way where I can disable the Code Highlighting of this extension?

I found another extension that has better code highlighting and formatting(https://github.com/CoffeeChaton/vscode-autohotkey-NekoHelp), but I didn't find any way to make their highlighter take precedence over Ahkpp.

For example, in the `settings.json` file I can set the default formatter to the another extension, I wonder if there is a way to do that with the `highlighter`  too:
![image](https://github.com/mark-wiemer/ahkpp/assets/20379136/e042e294-c278-4bd8-b50a-c53590ba8f31)

**Comment by kyklish** [comment]
Sorry for offtop. When AHK++ is better then NikoHelp? NikoHelp's description very impressive.

**Comment by mark-wiemer** [comment, marked as answer]
Currently there isn't a way to disable just the code highlighting. Unfortunately that feature isn't configurable like formatting, as far as I'm aware. If you find an open issue for this, please upvote it. Otherwise, you're welcome to create a new issue for this :)

**Reply by mark-wiemer** [reply]
#84 already covers this :)
